### DISCUSSION TITLE
gfx1032 support for python-pytorch-rocm

### DISCUSSION THREAD

**Discussion by acxz** [rocm-arch/rocm-arch#969] [Support]
From @GUUser91:
> Is it possible for python-pytorch-rocm to support gfx1032? This issue linked below says it you can use gfx1032 provided you add `export HSA_OVERRIDE_GFX_VERSION=10.3.0` to a .bashrc file [RadeonOpenCompute/ROCm#1698](https://github.com/RadeonOpenCompute/ROCm/issues/1698)

**Comment by L0grus** [comment]
I have gfx1032 (rx6650) and, unfortunately , export HSA_OVERRIDE_GFX_VERSION=10.3.0  does not helps. Pytorch-rocm says hipErrorNoBinaryForGpu: Unable to find code object for all current devices!
I also noticeed, there are no .kdb files in  /opt/rocm/share/miopen/db , but copying  gfx1030_36.kdb from amd repository pointless.

**Reply by tpkessler** [reply]
That's a ROCm bug, see the Arch bug tracker for more details.

**Reply by acxz** [reply]
Do you mind linking it the bug on the Arch bug tracker for the sake of completeness?
